lib/helper.rb

```diff
@@ -99,6 +99,12 @@ def print_horizontal_rule
 
   def replace_filename_special_chars(str, invalid_replace = false)
     result = str.tr("/:*?\"<>|.", "／：＊？”〈〉｜．").gsub("\\", "￥").gsub("\t", "").gsub("\n", "")
+    if Inventory.load("local_setting")["normalize-filename"]
+      begin
+        result.unicode_normalize!
```

Comment:
__[rubocop]__
* Lint/HandleExceptions: Do not suppress exceptions. [:ref](https://github.com/bbatsov/ruby-style-guide#dont-hide-exceptions)

<!-- Comment by [SideCI](https://sideci.com) -->